### DISCUSSION TITLE
Is it possible to achieve type safety?

### DISCUSSION THREAD

**Discussion by guilhermeg2k** [originjs/vite-plugin-federation#428] [Q&A]
I'm new on the mod fed world and one of my first question when i saw the concept it was about types. Doing a little research about it i found that webpack mod fed has that plugin https://github.com/module-federation/universe/tree/main/packages/typescript
Have searched something similitar to the vite plugin but found nothing, is it possible to achieve something similitar to that with this vite plugin?

**Comment by PaRoxUs** [comment]
Looking into this too, anyone has an insight into this?
